src/spikeinterface/extractors/tests/test_neoextractors.py

```diff
@@ -278,7 +278,6 @@ class CedRecordingTest(RecordingCommonTestSuite, unittest.TestCase):
     ]
 
 
-@pytest.mark.skipif(ON_GITHUB, reason="Maxwell plugin not installed on GitHub")
```

Comment:
Should we still spike them locally if the user does not have the plugin? Otherwise we will have fails when we run the tests locally.

Reply:
It auto-installs if needed

Reply:
Ah great!